server/views/pages/overview/partials/supportNeedsTabContents.njk

```diff
@@ -5,40 +5,58 @@
 
       <div class="govuk-summary-card" id="health-and-support-needs-summary-card">
         <div class="govuk-summary-card__title-wrapper">
-          <h2 class="govuk-summary-card__title">Health and support needs</h2>
+          <h2 class="govuk-summary-card__title">LDD and health needs</h2>
         </div>
         <div class="govuk-summary-card__content">
-          <dl class="govuk-summary-list">
+          <p class="govuk-hint">Information from Curious.</p>
 
-            <div class="govuk-summary-list__row">
-              <dt class="govuk-summary-list__key">
-                Language support needed
-              </dt>
-              <dd class="govuk-summary-list__value">
-                <ul class="govuk-list">
-                  {% for healthAndSupportNeed in supportNeeds.healthAndSupportNeeds %}
-                    <li>{{ healthAndSupportNeed.languageSupportNeeded }}</li>
-                  {% endfor %}
-                </ul>
-              </dd>
-            </div>
+          {% for healthAndSupportNeed in supportNeeds.healthAndSupportNeeds %}
 
-            <div class="govuk-summary-list__row">
-              <dt class="govuk-summary-list__key">
-                LDD and health needs
-              </dt>
-              <dd class="govuk-summary-list__value">
-                {% for healthAndSupportNeed in supportNeeds.healthAndSupportNeeds %}
-                  <ul class="govuk-list">
-                    {% for lddHealthNeed in healthAndSupportNeed.lddAndHealthNeeds %}
-                      <li>{{ lddHealthNeed }}</li>
-                    {% endfor %}
-                  </ul>
-                {% endfor %}
-              </dd>
-            </div>
+            <dl class="govuk-summary-list">
+              <h3>{{ healthAndSupportNeed.prisonName }}</h3>
```

Comment:
Minor but..

```
Error: Element [h3](https://html.spec.whatwg.org/multipage/#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements) not allowed as child of element [dl](https://html.spec.whatwg.org/multipage/#the-dl-element) in this context. (Suppressing further errors from this subtree.)
```

Reply:
Infact we could just move the `<h3>` above the `<dl>` and should be good?

Reply:
Ah, OK, so strictly speaking my markup is not quite right?
Let me see what I can do ...

Reply:
@chrimesdev - I've moved the h3 out of the dl (and also added some visually hidden text to aid accessibility)
Have a look and see what you think 🤞